packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -1299,7 +1299,7 @@ class _CupertinoActionSheetState extends State<CupertinoActionSheet> {
 
     final List<Widget> children = <Widget>[
       Flexible(
-        child: ClipRRect(
+        child: ClipRSuperellipse(
```

Comment:
Should we apply this to _buildCancelButton as well? It may look odd if it's not using the same shape.